Replicator/Replicator.cc

```diff
@@ -817,6 +820,7 @@ namespace litecore { namespace repl {
             MessageIn *response = progress.reply;
 
             if (response->isError()) {
+                logError("Error response from remote for request 'getCollections'");
```

Comment:
Maybe add error code and domain as well? It seems missing the information here.

Reply:
The code and domain of the response's error is already logged in `gotError()`, so I wanted to let that function handle error reporting, rather than bloating up this function. I just wanted to add some context as to where the (next line in the log) error came from.